### DISCUSSION TITLE
[PvP] Release Season 3 PvP after the release of Black Temple before the release of Zul'Aman

### DISCUSSION THREAD

**Discussion by Belodina** [chromiecraft/chromiecraft#7339] [Suggestions]
In an attempt to renew the state of PvP, and provide slightly competitive rewards in comparison to the gear from Black Temple.

This deviates from the original timeline of TBC, with Season 3 originally being released during patch 2.3, this change will further benefit the server and reward the PvP community.

**Comment by Amrx2** [comment]
Hi Belodina, just popping this there that whilst we won't be able to drop Season 3 alongside the release of Black Temple, we are hopefully going to be in a position to release it shortly after alongside some other work done in the PvP space. Thanks.
